app/src/main/java/de/westnordost/streetcomplete/quests/ferry/AddFerryAccessPedestrian.kt

```diff
@@ -0,0 +1,29 @@
+package de.westnordost.streetcomplete.quests.ferry
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.SimpleOverpassQuestType
+import de.westnordost.streetcomplete.data.osm.changes.StringMapChangesBuilder
+import de.westnordost.streetcomplete.data.osm.download.OverpassMapDataDao
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment
+
+class AddFerryAccessPedestrian(o: OverpassMapDataDao) : SimpleOverpassQuestType<Boolean>(o) {
+
+    override val tagFilters = "ways, relations with route=ferry and !foot"
+    override val commitMessage = "Specify ferry access"
+    override val icon = R.drawable.ic_quest_ferry
```

Comment:
A different icon (same icon but with pedestrian-blue background) would be nice.

Reply:
Should I copy `StreetComplete/app/src/main/res/drawable/ic_quest_ferry.xml` and modify background color or should I modify some generator of svg icons?

I see that in some commits also `StreetComplete/res/quest_icons.svg` was changed but it is unclear to me what is original source and how generation is triggered.

I found `export_svgs.py` script but it has note 

> # This script does not work, currently.

Reply:
Yes, you can simply copy ic_quest_ferry

Reply:
Done!